### DISCUSSION TITLE
Invalidating specific items

### DISCUSSION THREAD

**Discussion by Killen7** [reduxjs/redux#4710] [Help]
Hi!

I was reading the documentation about how to [invalidate a specific cache item](https://redux.js.org/tutorials/essentials/part-8-rtk-query-advanced#invalidating-specific-items), but it's not what I need.

Currently, my edit endpoint already returns the modified item, and I want to use that response to update the cached data without making a new call.

Any suggestions? Thanks!

**Comment by EskiMojo14** [comment]
See [Manual Cache Updates](https://redux-toolkit.js.org/rtk-query/usage/manual-cache-updates).
